students/paul_cooper/lesson_2/l2_grid_printer_p3.py

```diff
@@ -0,0 +1,15 @@
+n = 4 #enter the nuber of nxn squares
+q = 5 #enter the nuber of dash lines in the squares
+
+def print_grid(n,q):
+	print('+',('- '*q +'+ ')*n)
+
+	for i in range(n):
+		for j in range(q):
+			print('|',('  '*q +'| ')*n)
+		print('+',('- '*q +'+ ')*n)
+
+
+
+	
+print_grid(n,q)
```

Comment:
These work well, but same feedback for these files as your fizz buzz file - please fix formatting.